apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php

```diff
@@ -238,14 +270,7 @@ function wpcom_global_styles_in_use() {
 
 	$user_cpt = WP_Theme_JSON_Resolver::get_user_data_from_wp_global_styles( wp_get_theme() );
 
-	if ( ! isset( $user_cpt['post_content'] ) ) {
-		do_action( 'global_styles_log', 'global_styles_not_in_use' );
```

Comment:
We are no longer triggering this action. Is that intentional? I think it was used to add some debug traces.

Reply:
Actually! Maybe it was intentional. 😄 

The same `not_in_use` is logged in the caller, just a bit later, by the `wpcom_global_styles_in_use()` function.

I think that this is the most appropriate behaviour, as it records whether GS is **currently** in use.

If I put this additional log back here, it would also record whether GS is (not) used by all GS posts on the site.
We can discuss if it's interesting to know, but it's different than what we were recording before this change. 🙂